[DOCS-14295] Split Fleet Automation docs into multiple topics (#36698)
DataDog/documentation: content/en/agent/fleet_automation/_index.md

Datadog Fleet Automation allows you to centrally govern and remotely manage Datadog Agents and OpenTelemetry (OTel) Collectors at scale to support your evolving observability needs.

{{< img src="/agent/fleet_automation/fleet-automation-main.png" alt="The Fleet Automation page showing a list of Agents with their versions, statuses, and enabled products." style="width:100%;" >}}

## Key capabilities

With Fleet Automation, you can:
- **[View Agent and OTel Collector configurations][6]** along with historical changes to confirm deployment updates and verify configuration consistency.
- **[Keep your fleet up to date][7]** by identifying and upgrading outdated Agent and OTel Collector versions.

> - **[Configure Datadog Agents][8]** to centralize setup and gain visibility into your environments faster.

Reviewer: Very minor thing, but I noticed that these two appear in the opposite order in the sidebar. Might be worth unifying the narrative we use to talk about these features?

Author: Oh yeah I like that! I'll go with the order from the UI (fleet view -> configure -> upgrade). So the sidebar is correct.

- **[Send a support flare remotely][9]**, reducing the time it takes to debug issues on an Agent or DDOT Collector.

## Set up Fleet Automation

> ### Prerequisites

Author: Todo: move prereqs to child pages (view vs. upgrade/configure has different requirements)

- [Remote Configuration][1] enabled for your organization
- Agent version 7.73 or later
- Linux VMs installed using the install script or Ansible Datadog Role, or Windows VMs

<div class="alert alert-info">
Remote management of Agents in containerized workloads is not supported.
</div>

### Enable Agent and OTel Collector configuration views

- The Agent and Datadog Distribution of OTel Collector (DDOT) configuration view is enabled by default in Agent 7.47.0 or later. To enable it manually, set `inventories_configuration_enabled` to `true` in your [Agent configuration file][2], or use the `DD_INVENTORIES_CONFIGURATION_ENABLED` environment variable.

> - The upstream OTel Collector configuration view is enabled by setting the [Datadog Extension][3] in your collector configuration file.

Reviewer: ```suggestion
- To enable the upstream OTel Collector configuration view, set the [Datadog Extension][3] in your collector configuration file.
```

- Agent integration configuration is enabled by default in Agent 7.49 or later. To enable it manually, set `inventories_checks_configuration_enabled` to `true` in your [Agent configuration file][2], or use the `DD_INVENTORIES_CHECKS_CONFIGURATION_ENABLED` environment variable.

### Fleet Automation API
Fleet Automation provides a public API that allows you to programmatically view and manage Datadog Agents at scale. For full endpoint details and usage examples, see the [Fleet Automation API documentation][4]. 

> **Note**: The Fleet Automation API does not support all Datadog Agent configuration capabilities.

Reviewer: ```suggestion
<div class="alert alert-info">
The Fleet Automation API does not support all Datadog Agent configuration capabilities.
</div>
```
For consistency with other notey guy 🙂

Author: Love it!
